phpBB/styles/prosilver/template/attachment.html

```diff
@@ -29,6 +29,14 @@
 		</dl>
 		<!-- ENDIF -->
 
+		{% if _file.S_AUDIO_FILE %}
+		<dl class="file">
+			<dt><audio class="file-audio-controls" controls="controls" preload="metadata"><source src="{{ _file.U_DOWNLOAD_LINK }}" type="{{ _file.MIMETYPE }}"></audio></dt>
+			{% if _file.COMMENT %}<dd><em>{{ _file.COMMENT }}</em></dd>{% endif %}
+			<dd>{% if _file.UPLOAD_ICON %}{{ _file.UPLOAD_ICON }} {% endif %}<a class="postlink" href="{{ _file.U_DOWNLOAD_LINK }}" download>{{ _file.DOWNLOAD_NAME }}</a> ({{ _file.FILESIZE }} {{ _file.SIZE_LANG }}) {{ _file.L_DOWNLOAD_COUNT }}</dd>
```

Comment:
yet again please fix the spacing here, also give the link a component class as well like so
```
		<dd class="c-file-actions">
			{% if _file.UPLOAD_ICON %}
				<span class="o-icon c-file-icon">{{ _file.UPLOAD_ICON }}</span>
			{% endif %}
			<a class="c-file-actions-link postlink" href="{{ _file.U_DOWNLOAD_LINK }}" download>{{ _file.DOWNLOAD_NAME }}</a>
			<span class="c-file-actions-meta"> ( {{ _file.FILESIZE }} {{ _file.SIZE_LANG }}) {{ _file.L_DOWNLOAD_COUNT }} )</span>
		</dd>
```
I am not sure about the icon I will need to look deeper into how this icon is working? and what its purpose is.. Why is this coming from the database and not just using the normal icon?